src/cli/commands/force/lightning/setupRequirementsUtil.ts

```diff
@@ -0,0 +1,15 @@
+import { RequirementList } from '@salesforce/lwc-dev-mobile-core/lib/common/Requirements';
+import { AndroidEnvironmentRequirements } from '@salesforce/lwc-dev-mobile-core/lib/common/AndroidEnvironmentRequirements';
+import { IOSEnvironmentRequirements } from '@salesforce/lwc-dev-mobile-core/lib/common/IOSEnvironmentRequirements';
+import { CommandLineUtils } from '@salesforce/lwc-dev-mobile-core/lib/common/Common';
+import { Logger } from '@salesforce/core';
+
+export function getPlatformSetupRequirements(
+    logger: Logger,
+    platform: string,
+    apiLevel?: string
+): RequirementList {
+    return CommandLineUtils.platformFlagIsAndroid(platform)
+        ? new AndroidEnvironmentRequirements(logger, apiLevel)
+        : new IOSEnvironmentRequirements(logger);
+}
```

Comment:
I made a helper function which may go away after the next round of refactoring in `core`. This code might stay if we don't use `Setup` or another base command to inherit from(which I'm leaning more into). For now though I need this method because all commands need the same set of requirements as `Setup`.